transbank/webpay/webpay_plus/request/__init__.py

```diff
@@ -48,11 +48,12 @@ def __eq__(self, other) -> bool:
             and self.buy_order == other.buy_order
 
 
-class MallTransactionCreateDetails(object):
+class MallTransactionCreateDetails:
     __details = []
 
     def __init__(self, amount: float, commerce_code: str, buy_order: str):
-        self.add(amount, commerce_code, buy_order)
+        mall_details = MallDetails(amount, commerce_code, buy_order)
+        self.__details = [mall_details]
```

Comment:
```suggestion
        mall_details = MallDetails(amount, commerce_code, buy_order)
        self.__details = [mall_details]
```

Maybe it is cleaner to create a clean() method that resets the __details array and calling that function before self.add(amount, commerce_code, buy_order) on the init method? :)

Reply:
what do you think @jalvaradosegura? cc @ffiebig

Reply:
Yeah sure! i also think is cleaner. I will implement it

Reply:
@gonzunigad The changes are ready 😄